paypal/driver.py

```diff
@@ -221,7 +242,7 @@ def GetExpressCheckoutDetails(self, token):
         response_dict = parse_qs(response)
         self.api_response = response_dict
         state = self._get_value_from_qs(response_dict, "ACK")
-        if not state in ["Success", "SuccessWithWarning"]:
+        if state not in ["Success", "SuccessWithWarning"]:
```

Comment:
Flake8 violation for me, turns out I'm using a more recent version. I'll revert these changes.